platforms/core-runtime/stdlib-java-extensions/src/main/java/org/gradle/util/internal/CollectionUtils.java

```diff
@@ -450,7 +450,8 @@ public static <T, C extends Collection<? super T>> C addAll(C t1, Iterable<? ext
      * @param <T> The element type of t1
      * @return t1
      */
-    @SuppressWarnings("unchecked")
+    @SafeVarargs
+    @SuppressWarnings("varargs")
```

Comment:
❓ Why is @SuppressWarnings("varargs") necessary here? There is no `System.arraycopy` here.